src/addon.cc

```diff
@@ -671,6 +683,7 @@ void Initialize(v8::Local<v8::Object> exports) {
 	NODE_SET_METHOD(exports, "setAircraftInitialPosition", SetAircraftInitialPosition);
 	NODE_SET_METHOD(exports, "transmitClientEvent", TransmitClientEvent);
 	NODE_SET_METHOD(exports, "requestSystemState", RequestSystemState);
+	NODE_SET_METHOD(exports, "FlightLoad", FlightLoad);
```

Comment:
I think it's better if all exported methods have the same casing in their names (`"FlightLoad"` -> `"flightLoad"`)

Reply:
I'll fix the export line.
the function itself is like you had started, with function name identical to the sdk method it wrapped